idunn/blocks/environment.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import requests
+
+from apistar import validators
+from .base import BaseBlock
+from idunn import settings
+
+logger = logging.getLogger(__name__)
+
+
+
+class AirQuality(BaseBlock):
+    BLOCK_TYPE = "Environment"
+
+    air_quality = validators.Object(allow_null=True)
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        print(es_poi.PLACE_TYPE)
+        if es_poi.PLACE_TYPE != 'admin':
+            return None
+        if 'city' in es_poi.get('zone_type') or es_poi.get('zone_type') == 'suburb':
+            air_quality = get_air_quality(es_poi.get('bbox'))
+        else:
+            return None
+
+        if not air_quality:
+            return None
+
+        return cls(
+            air_quality=air_quality
+        )
+
+
+def get_air_quality(geobbox):
+    kuzzle_address = settings.get('KUZZLE_CLUSTER_ADDRESS')
+    kuzzle_port = settings.get('KUZZLE_CLUSTER_PORT')
```

Comment:
this function should be defined in `/idunn/api/kuzzle.py` to reuse the existing configuration